django/db/migrations/loader.py

```diff
@@ -186,15 +186,20 @@ def build_graph(self):
         # This is just for dependency re-pointing when applying replacements,
         # so we ignore run_before here.
         reverse_dependencies = {}
-        for key, migration in normal.items():
+        reverse_dependencies_normal = {}
+        for key, migration in self.disk_migrations.items():
             for parent in migration.dependencies:
                 reverse_dependencies.setdefault(parent, set()).add(key)
+                if key in normal:
+                    reverse_dependencies_normal.setdefault(parent, set()).add(key)
         # Remember the possible replacements to generate more meaningful error
         # messages
         reverse_replacements = {}
         for key, migration in replacing.items():
             for replaced in migration.replaces:
                 reverse_replacements.setdefault(replaced, set()).add(key)
+        # Set to store all replacing migrations that cannot be applied.
```

Comment:
I agree about the naming. How about "unapplicable" or "nonapplicable"? These words are much less popular in the english language, though they seem to better emphasise that this is a set of replacing migrations that cannot be applied in this scenario, rather than migrations that don't apply due to relevance or appropriateness. Could also use "not_applicable"? Thoughts?